src/utils/index.coffee

```diff
@@ -189,15 +204,29 @@ findKnownImportInDirectory = ({
     context
   }
   return null unless found = directoryCache.get name
-  {relativePath, type} = found
-  importPath = "#{prefix}#{relativePath}"
+  {prefixRelativePath, fullPath, type} = found
+  filename = context.getFilename()
+  importPath = if settings['known-imports/relative-paths']
+    relativePath = pathModule.relative pathModule.dirname(filename), fullPath
+    extension = pathModule.extname relativePath
+    ensureLeadingDot(
```

Comment:
The Nodejs `path` utilities were generating paths without a leading `./`, but I think it's at least a good starting point/default behavior to always include the leading `./` or `../` when generating relative paths